src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -1208,6 +1211,27 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
 
                 var changes = 0L;
 
+                foreach (var (docId, items) in _deletedRanges)
```

Comment:
I can't think of a scenario right now, but I wonder if it is a problem that we do things out of order here.
In the import we first delete and then append, regardless to the order of events in the original database.

Reply:
yes I thought about that too
If it was the other way around - on import first append and then delete - then I can
find a scenario where we would fail
but when we first delete and then append - couldn't think of a scenario that will cause problems